test/types/fetch.test-d.ts

```diff
@@ -17,7 +18,8 @@ import {
   Response,
   ResponseInit,
   ResponseType,
-  ReferrerPolicy
+  ReferrerPolicy,
+  ControlledAsyncIterable
```

Comment:
What is `ControlledAsyncIterable` and do we even need it?

Reply:
Not sure. This does not seem to be used or tested.

Reply:
I suggest you simply remove it...

Reply:
Done 👍

Reply:
When the types were published I don't think @types/node had a ReadableStream type, so this pseudo-readablestream one was added.